packages/connect-button/src/components/connect-button.ts

```diff
@@ -188,6 +193,48 @@ export class ConnectButton extends LitElement {
         : this.initialBodyOverflow
   }
 
+  private toggleParentBackdropFilter() {
+    const OPACITY_TRANSITION_DURATION = 180
+    if (!this.isMobile) return
+
+    if (!this.showPopoverMenu && this.parentElementWithBackdropFilter) {
```

Comment:
can we put this in a variable and name it to be something descriptive of the action?